db/sample_data/email_templates.rb

```diff
@@ -0,0 +1,60 @@
+def email_template_acceptance
+  EmailTemplate.new(
+    status: :acceptance,
+    hide_recipients: false,
+    subject: "Zusage für TODO",
```

Comment:
TODO schould be changes by the organizer (enter camp name, etc.) This is intended :)